deploy/utils/tests/test_convert_api_version.py

```
@@ -0,0 +1,447 @@
+    alpha_path = REPO_ROOT / f"examples/backends/vllm/deploy/{name}.yaml"
+    beta_path = REPO_ROOT / f"examples/backends/vllm/deploy/v1beta1/{name}.yaml"
+    if not alpha_path.exists() or not beta_path.exists():
+        pytest.skip(f"missing example pair for {name}")
+
+    converted = c.convert_docs(
+        c.load_docs(alpha_path.read_text()), target="nvidia.com/v1beta1"
+    )
+    expected = c.load_docs(beta_path.read_text())
+
+    assert len(converted) == len(expected)
+    for got, want in zip(converted, expected):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)